alphagov/tech-docs-gem
# Upgrade govuk-frontend and the crown logo

Closes  #337 #343

## What’s changed

- Update govuk-frontend from v4.4.1 to v4.7
- Update crown SVG and favicon to Tudor crown
- Manually adjust SVG crown with styles from govuk-frontend v4.8
- Suppress IE8 mixin deprecation warning introduced in govuk-frontend v4.6


Ideally we would upgrade to govuk-frontend v4.8 and not have to bring in the minor manual SVG alignment, but as govuk-frontend v4.8 introduced SCSS source maps, the tech-docs gem compiler includes them but not any styles.


## Identifying a user need

We need to update the crown before March 1

## How to test with your tech docs

To see how your changes to the gem affect your website, point your website's Gemfile to your local checkout of this branch:

```rb
gem 'govuk_tech_docs', path: '../tech-docs-gem'
```
